tools/errortracker/errortracker.go

```diff
@@ -174,9 +175,15 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	exception := r.URL.Query().Get("s")
+	// If format does not end with :\d+ truncate up to the last newline.
+	if !regexp.MustCompile(`:\d+\n?$`).MatchString(exception) {
+		exception = string(regexp.MustCompile(`\n.*$`).ReplaceAllString(exception, ""))
```

Comment:
Just a sanity check, are you sure this will match the very last `\n`? I keep stumbling on those greedy/not-greedy expressions.